string.c

```diff
@@ -420,90 +420,65 @@ str_store_precomputed_hash(VALUE str, st_index_t hash)
 }
 
 struct fstr_update_arg {
-    VALUE fstr;
     bool copy;
     bool force_precompute_hash;
 };
 
-static int
-fstr_update_callback(st_data_t *key, st_data_t *value, st_data_t data, int existing)
-{
-    struct fstr_update_arg *arg = (struct fstr_update_arg *)data;
-    VALUE str = (VALUE)*key;
-
-    if (existing) {
-        /* because of lazy sweep, str may be unmarked already and swept
-         * at next time */
+static VALUE build_fstring(VALUE str, struct fstr_update_arg *arg) {
```

Comment:
Would you align styles with the rest?

```suggestion
static VALUE
build_fstring(VALUE str, struct fstr_update_arg *arg)
{
```